index.js

```diff
@@ -0,0 +1,214 @@
+        document.querySelector('textarea').selectionEnd = cursor - 1;
+      }
+    }
+    if (item.code === 'Delete') {
+      document.querySelector('textarea').value = document.querySelector('textarea').value.slice(0, cursor) + document.querySelector('textarea').value.slice(cursor + 1, document.querySelector('textarea').value.length);
+      document.querySelector('textarea').selectionEnd = cursor;
+    }
+    if (item.code === 'ArrowLeft') if (cursor !== 0) document.querySelector('textarea').selectionEnd = cursor - 1;
+    if (item.code === 'ArrowRight') document.querySelector('textarea').selectionStart = cursor + 1;
+    if (item.code === 'ArrowUp') document.querySelector('textarea').selectionEnd = 0;
+    if (item.code === 'ArrowDown') document.querySelector('textarea').selectionStart = document.querySelector('textarea').value.length;
+    if (item.code === 'Tab') {
+      document.querySelector('textarea').value = `${document.querySelector('textarea').value.slice(0, cursor)}    ${document.querySelector('textarea').value.slice(cursor, document.querySelector('textarea').value.length)}`;
+      document.querySelector('textarea').selectionEnd = cursor + 4;
+    }
+    if (item.type === 'write') {
+      write(item);
+    }
+  } else if (item.code === 'Backspace' || item.code === 'Delete' || item.code === 'Tab' || item.code === 'Enter' || item.type === 'write') {
+    cursor = document.querySelector('textarea').selectionStart;
+    document.querySelector('textarea').value = document.querySelector('textarea').value.slice(0, document.querySelector('textarea').selectionStart) + document.querySelector('textarea').value.slice(document.querySelector('textarea').selectionEnd, document.querySelector('textarea').value.length);
+    document.querySelector('textarea').selectionStart = cursor;
+    document.querySelector('textarea').selectionEnd = cursor;
+    if (item.code !== 'Backspace' && item.code !== 'Delete') { enterSymbols(item); }
+  }
+}
+
+
+// PREPARATION
+
+function keyFill(language) {
+  localStorage.languageOfKeyboard = language;
+  document.querySelector('.keyboard').innerHTML = '';
+  objOfKeys.forEach((item) => {
+    const key = document.createElement('div');
+    key.className = `${item.size} ${item.code}`;
+    if (language === 'eng') {
+      if (item.view === 'double') {
+        key.innerHTML = `<p class="upLetter">${item.shiftEngValue}</p>`;
+        key.innerHTML += `<p class="centerLetter">${item.engValue}</p>`;
+      } else key.innerHTML = `<p class="centerLetter">${item.shiftEngValue}</p>`;
+    } else if (item.view === 'double') {
+      key.innerHTML = `<p class="upLetter">${item.shiftRuValue}</p>`;
+      key.innerHTML += `<p class="centerLetter">${item.ruValue}</p>`;
+    } else key.innerHTML = `<p class="centerLetter">${item.shiftRuValue}</p>`;
```

Comment:
Instead of lines 84-92 you can use:
![image](https://user-images.githubusercontent.com/31600165/79695759-4fd56a80-8281-11ea-8bb2-6a4e5d6cd88c.png)
 It allows you to use one code for both languages